Frontend/src/pages/ExploreSystems.js

```diff
@@ -160,7 +160,7 @@ function ExploreSystems() {
               }))
         } catch (error) {
             setSystems(null);
-            reportMajorError("Error: CANNOT CONNECT TO DATABASE: Make sure Docker is running correctly");
+            reportMajorError("Error: Issue fetching requested data");
```

Comment:
Can we make it more clear, something like..
```suggestion
            reportMajorError("There was an error while fetching the information requested. Please contact the system administrator.");
```